app/models/work/additional_credit.rb

```diff
@@ -26,6 +27,7 @@ class AdditionalCredit
     attr_json :name, :string
 
     def display_as
+      return "Translated by #{name}" if role == 'translator'
```

Comment:
D you need this special case "return" here? Why isn't the same mechanism already being used to display phographer credit (ie, the next line, `"#{role.humanize} #{name}"`) sufficient?

Reply:
The front end is supposed to be shown as "Translated by"... so we can do this, or change the name of the `role` on the actual model to `translated_by`. Granted, that would be more consistent with `photographed_by`, but for metadata purposes I just like the single noun "Translator".

Reply:
Good idea re: using the i18n -- the early `return` is kinda icky.

Reply:
I would just change the key to `translated_by` honestly. Becuase it's simple, and consistent with the existing `photographed_by`, I don't think it matters too much either way beyond a "preference", why not leave them consistent? 

Is it actually spec'd somewhere by stakeholders that front-end should say "translated by"? I guess that makes sense though, again cause it's consistent with "photgraphed by"!

Reply:
OK never mind -- I'm capitulating and using `translated_by` as the label.